src/main/java/org/ow2/proactive/catalog/report/CellFactory.java

```diff
@@ -0,0 +1,138 @@
+
+    private static final String LIGHT_CYAN = "#F3F3F4";
+
+    private static final String WHITE = "#ffffff";
+
+    private static final String BLACK = "#000000";
+
+    public void addMainTitleCell(Row<PDPage> row, String data) {
+        createDataCell(row,
+                       (100 / 12f) * 9,
+                       data,
+                       MAIN_TITLE_FONT_SIZE,
+                       HorizontalAlignment.CENTER,
+                       VerticalAlignment.MIDDLE,
+                       ACTIVEEON_ORANGE,
+                       ACTIVEEON_BLUE);
+    }
+
+    public void addSecondaryHeaderCell(Row<PDPage> row, String data) {
+        createDataCell(row, 100, data, HorizontalAlignment.CENTER);
+    }
+
+    public void createDataCell(Row<PDPage> row, float width, String data) {
+        createDataCell(row, width, data, MEDIUM_FONT);
+    }
+
+    public void createDataHeaderCell(Row<PDPage> row, float width, String title) {
+        createDataCell(row,
+                       width,
+                       title,
+                       BIG_FONT,
+                       HorizontalAlignment.CENTER,
+                       VerticalAlignment.TOP,
+                       LIGHT_CYAN,
+                       BLACK);
+    }
+
+    public void createDataCellBucketName(Row<PDPage> row, float width, String data) {
+        createDataCell(row, width, data, BIG_FONT, HorizontalAlignment.CENTER, VerticalAlignment.MIDDLE, WHITE, BLACK);
+    }
+
+    public void createKeyValueContentDataCell(Row<PDPage> row, float width, String data) {
+        createDataCell(row, width, data, SMALL_FONT, HorizontalAlignment.LEFT, VerticalAlignment.TOP, WHITE, BLACK);
+    }
+
+    public void createDataCell(Row<PDPage> row, float width, String data, int fontSize) {
+        createDataCell(row, width, data, fontSize, HorizontalAlignment.CENTER, VerticalAlignment.TOP, WHITE, BLACK);
+    }
+
+    public void createDataCell(Row<PDPage> row, float width, String data, HorizontalAlignment align) {
+        createDataCell(row, width, data, MEDIUM_FONT, align, VerticalAlignment.TOP, WHITE, BLACK);
+    }
+
+    public void createIconCell(Row<PDPage> row, float width, String url_path) {
+        try {
+            URL url = new URL(url_path);
+            BufferedImage imageFile = ImageIO.read(url);
+            row.createImageCell(width, new Image(imageFile));
+        } catch (Exception e) {
+            createDataCell(row,
+                           width,
+                           url_path,
+                           MEDIUM_FONT,
+                           HorizontalAlignment.LEFT,
+                           VerticalAlignment.TOP,
+                           LIGHT_GRAY,
+                           BLACK);
+        }
+    }
+
+    private void createDataCell(Row<PDPage> row, float width, String data, int fontSize, HorizontalAlignment align,
+            VerticalAlignment valign, String fillColor, String textColor) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused method parameter "valign". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1172)